### ISSUE TITLE
interactive event: mouseover for icons not visible

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3359]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/interaktive-veranstaltung-11/
**user:** moderator/ user
**expected behaviour:** mouseover for all icons
**behaviour:** mouseover only for a few of them
**important screensize:**
**device & browser:** tested on mac, chrome
**Comment/Question:** 

Screenshot?

**Comment by CarolingerSeilchenspringer** [comment]
if easy, please include in this milestone. if complicated please come back to me.
